framework/lib/components/createable-select-dropdown/createable-select-dropdown.tsx

```diff
@@ -125,29 +135,31 @@ export const CreatableSelectDropdown = <T extends string = string> ({
     },
   })
 
-  const handleInputChange = (newValue: string) => {
-    setNewOptionText(newValue)
-  }
-
-  const handleCreateOption = () => {
-    const newOptionTextLower = newOptionText.toLowerCase()
-    const newOptionExists = standardOptions.some((option) =>
-      option.value.toLowerCase() === newOptionTextLower)
+  const handleCreateOption = (newOptionTextString: string = '') => {
+    const newOptionTextLower = newOptionTextString.toLowerCase()
+    const newOptionExists = standardOptions.some(
+      (option) => option.value.toLowerCase() === newOptionTextLower
+    )
```

Comment:
I think this bug was here before, but you want to check if the newOptionTextString is included in `createdOptions` array as well, otherwise you could create the same option twice